package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clr-addons",
-  "version": "19.2.0",
+  "version": "19.2.1",
```

Comment:
pls update to "19.3.0"
also execute following:
- "npm i" to update package-lock
- "npm i" in website folder to update package-lock